app/views/social_accounts/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1><%=  I18n.t 'social_accounts.edit.heading' %></h1>
+
+<%= form_with(model: [@user, @social_account], local: true) do |form| %>
```

Comment:
still not sure why the form partial isn't rendered here

Reply:
Good point.